form/src/main/java/org/dhis2/form/ui/provider/inputfield/ImageInputProvider.kt

```diff
@@ -48,6 +48,7 @@ internal fun ProvideInputImage(
             uploadState = UploadState.UPLOADING
             fieldUiModel.invokeUiEvent(UiEventType.ADD_PICTURE)
         },
+        onImageClick = {},
```

Comment:
this and the equivalent in SignatureProvider will be removed when  issue https://dhis2.atlassian.net/jira/software/c/projects/ANDROAPP/boards/113?selectedIssue=ANDROAPP-5587 is merged